server/src/policies/permissions.config.ts

```diff
@@ -0,0 +1,142 @@
+export type AccessType = 'ministry' | 'tc' | 't3ch' | 'admin' | 'stewardship';
+
+export interface PermissionConfig {
+  requiresAuth: boolean;
+  allowedAccessTypes?: AccessType[];
+  description?: string;
+}
+
+// Centralized permissions configuration
+/**
+ * By default, there is a hierarchy in this accessTypes
+ * Admin > Stewardship > tc > t3ch > ministry > isLoggedIn
+ * But this permission schema will not follow the ones in the main web.
+ * It will use the more conventional approach, i.e. only the access types listed can access it. 
+ * Please make sure the proper access level are listed.
+ */
+// TODO: Confirm on which access types can do what
+export const ROUTE_PERMISSIONS: Record<string, PermissionConfig> = {
```

Comment:
lets actually review all the permision config together, think its been a while since we touched this